test/unit/test-geodata.cpp

```diff
@@ -327,9 +326,8 @@ TEST_CASE("Default IOData", "[iodata][Serial]")
   iodata.domains.materials.back().attributes = {0};
 
   // Pull from the mfem externals data folder.
-  auto ref_tet_path = fs::path(MFEM_DATA_PATH) / "ref-tetrahedron.mesh";
-  mfem::Mesh single_tet(ref_tet_path.string());
-  mfem::ParMesh pmesh(Mpi::World(), single_tet);
+  mfem::Mesh mesh = mfem::Mesh::MakeCartesian3D(1, 1, 1, mfem::Element::TETRAHEDRON);
```

Comment:
I changed the mesh here because it doesn't really matter what mesh we use, and I preferred picking something that does not depend on a file.